src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -10,19 +10,34 @@
     private int figureLeg2 = random.nextInt(10);
     private int figureLeg3 = random.nextInt(10);
```

Comment:
if the circle will be produced - we'll store two unnecessary variables. we don't need these `figureLegs`